src/ansible_navigator/actions/config.py

```diff
@@ -321,7 +321,7 @@ def _parse_and_merge(self, list_output, dump_output) -> None:
                         parsed[variable]["via"] = source
                     parsed[variable]["current"] = current
                     parsed[variable]["__current_value"] = extracted.groupdict()["current"]
-                except KeyError as exc:  # noqa: F841
+                except KeyError as _exc:  # noqa: F841
```

Comment:
Wouldn't "fixing" the variable mean that `noqa` is no longer needed?

> **F841:** Local variable name is assigned to but never used

Reply:
yeah, brb

Reply:
Perhaps, the real fix is
```suggestion
                except KeyError:
```